blockchain/block/header.go

```diff
@@ -92,6 +106,7 @@ func (h *Header) BlockHeaderCoreProto() *iotextypes.BlockHeaderCore {
 	header := iotextypes.BlockHeaderCore{
 		Version:          h.version,
 		Height:           h.height,
+		GasUsed:          h.gasUsed,
```

Comment:
It should be a HF, which affecting block hash?

Reply:
we need to test it

Reply:
it will be 0 (default value of protobuf) for past blocks, so should be backward-compatible